nmrfx-processor/src/main/java/org/nmrfx/processor/processing/SampleSchedule.java

```diff
@@ -402,12 +402,7 @@ public VecIndex getNextGroup(final int vecGroup, MultiVecCounter tmult) {
         }
         if (k < nSamples) {
             int[] oSizes;
-            if (demo) {
-                oSizes = tmult.getOutSizes();
-            } else {
-//                oSizes = outMult.getOutSizes();
-                oSizes = tmult.getOutSizes();
-            }
+            oSizes = tmult.getOutSizes();
```

Comment:
Collapsed the if loop but not sure if this is correct, looks like outMult.getOutSizes was commented out to just use tMult, if this change is correct the function comment needs to be updated as it contradicts what is happening (https://github.com/nanalysis/nmrfx/blob/7d9b8e15b120fd9ddb133501a913dcd9801e67fe/nmrfx-processor/src/main/java/org/nmrfx/processor/processing/SampleSchedule.java#L383)

Maybe remove everything in the comment after the first sentence?

Reply:
It looks like this method isn't even used. I'd remove it (as well as other unused methods and commented code, in general). We need to be careful because some methods can be used from python script and are not detected by tools such as intellij, but I think this one is safe?

Reply:
Removed this method